### ISSUE TITLE
Switch to FastNEAR Indexer

### ISSUE THREAD

**Issue by volovyks** [near/mpc#937]
### Description
- Determine if any code updates are required
- Change the URL

**Comment by ppca** [comment]
TLDR: nontrivial code changes but save hassle from lake, however can be expensive. Likely 2 wk work.

1. there's no lake equivalent for FastNear, they have https://github.com/fastnear/libs/tree/main/neardata-fetcher
2. according to the example: https://github.com/fastnear/clickhouse-provider/blob/main/src/main.rs#L149, it seems like what we need to do to use the neardata-fetcher is start the data fetcher, listen to the receiver of fetcher and process each BlockWithTxHashes as they come in, kinda reusing handle_block() logic. 
4. fetcher can start from random heights as well, and we could extract the same information we had before from fetcher. 
5. Using FastNear can reduce the near lake issues we've been seeing and remove dependency on the near s3. This would be where we want to head.
6. Cost can be really high. If the fetcher does 3 fetches per second, then each month each instance of fetcher will do 8M requests. We currently have 12 dev nodes, 1 mainnet node, 1 testnet node, then we are at 8 * 14 = 112 M, which is even higher than the business plan at $499/month: https://fastnear.com/#pricing. I'll modify with updated info if any
